### PR TITLE
add satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1

### PR DESCRIPTION
**Pull request by puripuri2100**
Name: siunitx
Package name: satysfi-siunitx
Site: https://github.com/puripuri2100/SATySFi-siunitx
Installation method: Satyrographos-0.0.1 Makefile

### CODE REVIEW

File: packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "satyrographos" {>= "0.0.1" & < "0.0.2"}
+]
+build: [ ]
+install: [
+    [make "-f" "Makefile" "install" "LIBDIR=%{share}%/satysfi"]
+]
+remove: [
+    [make "-f" "Makefile" "uninstall" "LIBDIR=%{share}%/satysfi"]
+]
+
```
[the remark this answers is not in the document]
Comment (puripuri2100) [reply packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam]: Added the following code.

```
url {
  git: "git+https://github.com/puripuri2100/SATySFi-siunitx/commit/933d607a4a8816f70199908a2f19e1d38fbdfa80"
}
```

File: packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "satysfi-siunitx"
+version: "0.1"
```
[the remark this answers is not in the document]
Comment (puripuri2100) [reply packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam]: fixed

**Comment by puripuri2100** [comment]
There was no error.

```
$ opam lint packages/satysfi-siunitx/satysfi-siunitx0.1+satysfi0.0.3+satyrograhos0.0.1/opam
packages/satysfi-siunitx/satysfi-siunitx0.1+satysfi0.0.3+satyrograhos0.0.1/opam: Passed.
```

File: packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam (hunk 1 above)
[the remark this answers is not in the document]
Comment (na4zagin3) [reply packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam]: Do you mean this?
```
url {
  url: "git+https://github.com/puripuri2100/SATySFi-siunitx.git#933d607a4a8816f70199908a2f19e1d38fbdfa80"
}
```

- `git` field is deprecated according to the [manual § `url:`](https://opam.ocaml.org/doc/Manual.html#urlfield-git)
- `git+https` schema requires a commit identifier after `#`. See [manual § URLs](https://opam.ocaml.org/doc/Manual.html#URLs)

File: packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam (hunk 2 above)
[the remark this answers is not in the document]
Comment (na4zagin3) [reply packages/satysfi-siunitx/satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1/opam]: Well, my language wasn’t clear enough.
What I meant was that “The field `version: "0.1"` contradicts the version string in the path `satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1`, therefore can you change the line `version: "0.1"` -> `version: "0.1+satysfi0.0.3+satyrograhos0.0.1"`?”

The directory name `satysfi-siunitx.0.1+satysfi0.0.3+satyrograhos0.0.1` was correct and so shouldn’t have been changed it.

**Comment by na4zagin3** [comment]
LGTM!

**Comment by na4zagin3** [comment]
Thanks!

**Pull request merged**